src/views/InteractiveMap/InteractiveMap.vue

```diff
@@ -457,59 +457,95 @@ export default Vue.extend({
           hasSimulationError: false
         }
       });
-      axios
-        .post(`${settings.apis.model}/simulate`, {
-          model_id: model.id,
-          method: card.method,
-          operations: operations,
-          objective_id: card.objective.reaction
-            ? card.objective.reaction.id
-            : model.default_biomass_reaction,
-          objective_direction: card.objective.maximize ? "max" : "min"
-        })
-        .then(response => {
-          const wildtypeFluxdistribution = response.data.flux_distribution;
-          // Calculate new bounds based on manipulation scores calculated by DiffFVA
-          // (reference flux * score) / production growth
-          const editedBounds = card.manipulations
-            .map(function(manipulation) {
-              const newBound = Math.round(
-                (wildtypeFluxdistribution[manipulation.id] *
-                  manipulation.value) /
-                  card.productionGrowthRate
-              );
+      // Apply the values from diffFVA results as bounds for simulation
+      const editedBounds = card.manipulations.map(function(manipulation) {
```

Comment:
Prefer arrow function here, otherwise `this` won't be available in the scope

```suggestion
      const editedBounds = card.manipulations.map(manipulation => {
```